src/react-components/settings-menu.js

```diff
@@ -151,8 +151,8 @@ export default class SettingsMenu extends Component {
               <a href="https://discord.gg/wHmY4nd" target="_blank" rel="noreferrer noopener">
                 <FormattedMessage id="settings.community" />
               </a>
-              <a href="/spoke" target="_blank" rel="noreferrer noopener">
-                <FormattedMessage id="settings.create-scene" />
+              <a target="_blank" href="https://forms.gle/wYyAowiQyK9EDdcDA" rel="noopener noreferrer">
```

Comment:
I would make the response field in this form a text box so that people can type more than 1 sentence:
![image](https://user-images.githubusercontent.com/4072106/55433480-53009a00-554a-11e9-9583-e7d0f3125a1c.png)

Reply:
cc @misslivirose

Reply:
The UI isn't great at displaying it, but it actually is a long-form answer for paragraph responses

Reply:
Ohh I see -- I didn't even try. My bad!